packages/frontend/package.json

```diff
@@ -11,8 +11,11 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "i18next": "^22.1.4",
+    "i18next-browser-languagedetector": "^7.0.1",
```

Comment:
YAGNI? This is something we won't be using right now, so I'd say that we should leave this out (maybe add a comment where you could use this) as it's just extra cruft at this point.